src/main/java/org/ga4gh/starterkit/wes/utils/requesthandler/SubmitRunRequestHandler.java

```diff
@@ -103,21 +107,25 @@ private void validateRunRequest() throws BadRequestException {
         // Validate workflowType
         // - assert requested workflowType is supported according to ServiceInfo
         if (!serviceInfo.isWorkflowTypeSupported(workflowType)) {
-            throw new BadRequestException(
```

Comment:
We should not remove any existing exceptions. I would just revert these changes. Its okay if we don't add a log here as we are not catching an exception.